pluginframework/common/converters_test.go

```diff
@@ -0,0 +1,256 @@
+package pluginframework
+
+import (
+	"context"
+	"fmt"
+	"reflect"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-framework/types"
+	"github.com/stretchr/testify/assert"
+)
+
+type DummyTfSdk struct {
+	Enabled           types.Bool                  `tfsdk:"enabled" tf:"optional"`
+	Workers           types.Int64                 `tfsdk:"workers" tf:""` // Test required field
```

Comment:
Is the `tf` option present on every field?

Reply:
Yes